WSL/wsl2-mount-disk.md

```diff
@@ -0,0 +1,110 @@
+---
+title: Mount disks in WSL2 (preview)
+description: Usage instructions to mount and attach physical disks in WSL2.
+keywords: BashOnWindows, bash, wsl, windows, windows subsystem for linux, windowssubsystem, windows 10, install, WSL2, version 2, disk, mount
+ms.date: 08/03/2020
+ms.topic: article
+ms.localizationpriority: high
+---
+
+# Mounting a disk using WSL 2 (preview)
```

Comment:
This content is very similar to the tutorials content. Can we move this to tutorials for now and then put this content into the official docs when it is a full part of Windows?